app/code/Magento/Fedex/Model/Carrier.php

```diff
@@ -724,6 +724,8 @@ public function getCode($type, $code = ''): \Magento\Framework\Phrase|array|fals
                 'STANDARD_OVERNIGHT' => __('Standard Overnight'),
                 'FEDEX_FREIGHT' => __('Freight'),
                 'FEDEX_NATIONAL_FREIGHT' => __('National Freight'),
+                'FEDEX_REGIONAL_ECONOMY' => __('Regional Economy'),
+                'FEDEX_INTERNATIONAL_PRIORITY_EXPRESS' => __('Internationa Priority Express'),
```

Comment:
typo here

Reply:
indeed - but not where its important - fixed the label!
There are many other rates available - the ones i added are helpful for UK merchants